src/sentinel.c

```diff
@@ -49,87 +49,145 @@ extern SSL_CTX *redis_tls_ctx;
 extern SSL_CTX *redis_tls_client_ctx;
 #endif
 
+/* sentinel 默认端口号 */
 #define REDIS_SENTINEL_PORT 26379
 
 /* ======================== Sentinel global state =========================== */
 
 /* Address object, used to describe an ip:port pair. */
+/* 地址对象，用于保存主机名称，IP地址和端口号 */
 typedef struct sentinelAddr {
     char *hostname;         /* Hostname OR address, as specified */
     char *ip;               /* Always a resolved address */
     int port;
 } sentinelAddr;
 
 /* A Sentinel Redis Instance object is monitoring. */
+/* 实例是一个主服务器 */
 #define SRI_MASTER  (1<<0)
+/* 实例是一个从服务器 */
 #define SRI_SLAVE   (1<<1)
+/* 实例是一个 Sentinel */
 #define SRI_SENTINEL (1<<2)
+/* 实例当前状态为:SDOWN */
```

Comment:
```suggestion
/* 实例当前状态为: SDOWN */
```
空一个格或者下面的ODOWN不空格，格式统一一下